pages/register.js

```diff
@@ -142,14 +146,14 @@ class Register extends Component {
                   <FontAwesomeIcon icon="lock" fixedWidth />
                 </label>
 
-                <input
+                <PasswordInput
```

Comment:
Password Input has no suggestions or warnings on by default..
to turn them on add the following props:
```
showWarnings
showSuggestions
showStrength
```

I kept these 3 features separate ~~because thats what we had on fr,.com~~ just in case we wanted to use any of these features separately.